.taskfiles/tracetest/Taskfile.yml

```diff
@@ -5,17 +5,28 @@ tasks:
   default:
     desc: "Configure tracetest"
     deps:
-      - install
+      - install-linux
+      - install-darwin
     cmds:
       - tracetest configure -g --endpoint http://localhost:11633
 
-  install:
+  install-darwin:
     internal: true
     desc: "Install tracetest"
     status:
       - type tracetest
+    platforms: [darwin]
     cmds:
-      - cmd: brew install kubeshop/tracetest/tracetest
-        platforms: [darwin]
-      - cmd: curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh | bash
-        platforms: [linux]
+      - brew install kubeshop/tracetest/tracetest@0.12.2
+
+  install-linux:
+    internal: true
+    desc: "Install tracetest"
+    status:
+      - type tracetest
+    platforms: [linux]
+    cmds:  # https://docs.tracetest.io/getting-started/cli-installation-reference#apt
+      - sudo apt-get update && sudo apt-get install -y apt-transport-https ca-certificates
```

Comment:
Is this for a dockerfile or something? Do they maybe have a binary release we could wget/extract?

Reply:
@titpetric it's literally their installation steps on https://docs.tracetest.io/getting-started/cli-installation-reference#apt